usaspending_api/api_contracts/contracts/v2/disaster/recipient/business_category/loans.md

```diff
@@ -89,6 +93,8 @@ Returns loan spending details of recipient business categories receiving supplem
 + `children` (optional, array[Result], fixed-type)
 + `count` (required, number)
 + `face_value_of_loan` (required, number, nullable)
++ `obligation` (required, number, nullable)
```

Comment:
This file could also use the `sort` options

Reply:
done